dashboard/lti_new.py

```diff
@@ -35,6 +35,13 @@
 COURSE_MEMBERSHIP = 'http://purl.imsglobal.org/vocab/lis/v2/membership'
 DUMMY_CACHE = 'DummyCache'
 
+# do not require deployment ids if LTI_CONFIG_DISABLE_DEPLOYMENT_ID_VALIDATION is true
+class ExtendedDjangoMessageLaunch(DjangoMessageLaunch):
+    def validate_deployment(self):
+        if settings.LTI_CONFIG_DISABLE_DEPLOYMENT_ID_VALIDATION:
+            return self
+
+        return super(ExtendedDjangoMessageLaunch, self).validate_deployment()
```

Comment:
super() in [Python 3 should do the same thing](https://docs.python.org/3/library/functions.html#super) as this.

```suggestion
        return super().validate_deployment()
```

Reply:
Fixed